docs/developer/images.md

```diff
@@ -0,0 +1,128 @@
+# Art and Images
+
+## Colors
+
+Images in arcade are stored using 4 bits per color. The color `0` is
+reserved for transparency, so there are 15 available colors for you to
+work with at any one time. Colors are represented as indices into a palette;
+the default palette shown in the MakeCode UI is here:
+
+```
+#000000     // transparency
+#ffffff
+#ff2121
+#ff93c4
+#ff8135
+#fff609
+#249ca3
+#78dc52
+#003fad
+#87f2ff
+#8e2ec4
+#a4839f
+#5c406c
+#e5cdc4
+#91463d
+#000000
+```
+
+
+### Changing the default palette
+
+If you intend to use the MakeCode blocks editor for
+your project but would like to change the colors, it is recommended
+that you set a default palette so that your sprites render correctly
+in the image editor/blocks.
+
+To change the default palette of a project, go into the project
+settings in the MakeCode editor (under the cogwheel in the upper right).
+Once there, click "Edit settings as text" and add a `"palette"` entry
+like so:
+
+```JSON
+{
+    "name": "Untitled",
+    "dependencies": {
+        "device": "*"
+    },
+    "description": "",
+    "files": [
+        "main.blocks",
+        "main.ts",
+        "README.md"
+    ],
+    "palette": [
+        "#000000",
+        "#ffffff",
+        "#ff2121",
+        "#ff93c4",
+        "#ff8135",
+        "#fff609",
+        "#249ca3",
+        "#78dc52",
+        "#003fad",
+        "#87f2ff",
+        "#8e2ec4",
+        "#a4839f",
+        "#5c406c",
+        "#e5cdc4",
+        "#91463d",
+        "#000000"
+    ]
+}
+```
+
+Note that the first color in the array doesn't matter because it maps to transparency.
+Be sure that the palette you enter has exactly 16 colors.
+
```

Comment:
Note that it doesn't matter what you set the first color in the array to because it always maps to transparency. Be sure that the palette array you enter has **exactly** 16 colors.